### PR TITLE
Update international telephone validation

### PR DESCRIPTION
**Pull request by ethax-ross**
### Trello card

[Trello-1597](https://trello.com/c/WCPR0FxQ/1597-overseas-number-validation-sanitisation)

### Context

When a candidate is overseas we need to ensure that their telephone number will be in a format that the TPUK dialling system supports. On the client-side this means the number needs to be prefixed by the country dial-in code (e.g. 44). The API will then sanitise the number further to remove non-numerical values, white space and a proceeding `+`.

### Changes proposed in this pull request

- Update international telephone validation

### Guidance to review

Screenshot 2021-06-02 at 11 07 02

### CODE REVIEW

File: app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <p>We will need to call you to discuss your qualifications. You must have the details of your overseas qualifications when we contact you.</p>
 <p>In the meantime, you can call us on Freephone <%= link_to("0800 389 2500", "tel://08003892500") %> between 8.30am and 5.30pm Monday to Friday.</p>
 
-<%= f.govuk_phone_field :address_telephone, width: 20 %>
+<%= f.govuk_phone_field :address_telephone, width: 20, prefix_text: "+" %>
```
Comment (peteryates) [inline app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb:6]: 🙌🏽

**Review by peteryates — APPROVED** [review]
Looks good to me.

**Review by MylesJarvis — DISMISSED** [review]
LGTM - we talked about validation and the difficulties we would experience in handling that. With that in mind, this is sufficient

**Review by peteryates — APPROVED** [review]
Looks good to me

**Pull request merged**